WordPress/Classes/Services/PostRepository.swift

```diff
@@ -312,10 +312,10 @@ final class PostRepository {
         // If the post is already in trash, do nothing. If the app were to
         // proceed with `/delete`, it would permanently delete the post.
         if remotePost.status != BasePost.Status.trash.rawValue {
-            remotePost = try await remote.trashPost(PostHelper.remotePost(with: post))
+            try await remote.deletePost(withID: postID)
```

Comment:
Just out of curiosity, calling `/delete` once puts it in the trash, and calling `/delete` a second time deletes it permanently?

Reply:
> Just out of curiosity, calling /delete once puts it in the trash, and calling /delete a second time deletes it permanently?

Yes, and that's how it always worked.

It's not exactly safe because with `/delete` you can relatively easily create a scenario (even on wp.com) where the app shows up as "published", but is already trashed, and you accidentally permanently delete it by tapping "Trash".

Btw, we briefly tried changing this approach and implement trashing in a more safe manner by patching `post.status` instead but that didn't work for some scenarios. I'm not sure why there is no context in https://github.com/wordpress-mobile/WordPress-iOS/pull/22987 but it is the revert.